docs/modules/nifi/pages/usage_guide/extra_volumes.adoc

```diff
@@ -0,0 +1,17 @@
+= Adding External Files to the NiFi Servers
+
+Since Apache NiFi allows executing pretty much arbitrary workflows depending on which processors are used, it may become necessary to add external files to the pods.
+These could for example be client certificates used to configure a `PollHTTP` processor, a keytab to obtain a Kerberos ticket, or similar things.
+
+In order to make these files available the operator allows specifying extra volumes that will be added to the NiFi pods.
+
+[source,yaml]
+----
+spec:
+  extraVolumes:
+  - configMap:
+      name: nifi-client-certs
+    name: certs
+----
+
+All `Volumes` specified in this sectino will be made available under `/stackabe/userdata/{volumename}`.
```

Comment:
I'm unsure how to make commits which would appear in this PR, so I'll comment:

Couple of typos: `sectino` and `stackabe`

Reply:
Just for your information: You can add a suggestion using the button below

![image](https://user-images.githubusercontent.com/29303194/222979931-5414a6fc-0f7b-4125-a8ad-9cb7e8880d6e.png)

Reply:
typos fixed

Reply:
typos fixed